src/Calendar.js

```diff
@@ -259,6 +259,17 @@ class Calendar extends React.Component {
      */
     resourceTitleAccessor: accessor,
```

Comment:
let's also uptick the version in package.json and deploy it to npm so we can download it on the web project

Reply:
maybe we should merge this in, and deploy from master